doc/source/parameters/dem/post-processing.rst

```diff
@@ -13,16 +32,14 @@ Lethe-DEM has built-in post-processing capabilities. All post-processing results
 .. important::
  The particles' average velocity and the granular temperature are cell based results calculated instantaneously at each outputting time-step. This means that they are only instantaneous spatial averages, i.e, DO NOT correspond to temporal averages measured between the outputs. For a higher resolution of the results in time, we recommend the use of a lower ``output frequency`` (:doc:`../dem/simulation_control`).
 
-The post-processing subsection of the ``.prm`` file is according to the following example:
+----------------------------
+Force chains post-processing
+----------------------------
+The ``force chains`` feature enables VTU, PTVU, and PVD files to visualize force chains between particles. Generated files's names begin with the prefix ``force_chains-``. The files include:
 
-.. code-block:: text
-
- subsection post-processing
-  # Enable output of grid, granular temperature, and particles' average velocity
-  set Lagrangian post-processing = false
- end
+* Lines (use wireframe display with paraview) to represent forces. 
 
-.. note::
- By default, the post-processing is set to ``false``.
+* The force values for each line.
 
-* The ``Lagrangian post-processing`` enables the built-in post-processing. The current outputs include the grid of the domain, the granular temperature, and the average velocity of particles (in x, y, z and its norm).
+.. important::
```

Comment:
```suggestion
.. warning::
```

Reply:
I don't thing it's super expensive. Forces are being computed at every time step. But maybe I'm wrong.

Reply:
No it's not that expensive that's for sure, I think making the triangulation and writing it is more expensive than the force calculation. I think that adding an image and maybe even a link to the youtube video would be a good idea